core/workspace_comment_svg.js

```diff
@@ -97,30 +102,60 @@ Blockly.WorkspaceCommentSvg = function(
    * @type {boolean}
    * @private
    */
-  this.useDragSurface_ =
-      Blockly.utils.is3dSupported() && !!workspace.blockDragSurface_;
+  this.useDragSurface_ = utils.is3dSupported() && !!workspace.blockDragSurface_;
 
-  Blockly.WorkspaceCommentSvg.superClass_.constructor.call(
+  WorkspaceCommentSvg.superClass_.constructor.call(
       this, workspace, content, height, width, opt_id);
 
   this.render();
 };
-Blockly.utils.object.inherits(
-    Blockly.WorkspaceCommentSvg, Blockly.WorkspaceComment);
+object.inherits(WorkspaceCommentSvg, WorkspaceComment);
 
 /**
  * The width and height to use to size a workspace comment when it is first
  * added, before it has been edited by the user.
  * @type {number}
  * @package
  */
-Blockly.WorkspaceCommentSvg.DEFAULT_SIZE = 100;
+WorkspaceCommentSvg.DEFAULT_SIZE = 100;
+
+/**
+ * Size of the resize icon.
+ * @type {number}
+ * @const
+ * @private
+ */
+WorkspaceCommentSvg.RESIZE_SIZE = 8;
```

Comment:
Since these are only used in this file, you can just declare them as `const RESIZE_SIZE = 8;` (and so on).

Reply:
sgtm. I'll take a another pass and update those.